src/tools/genpolicy/src/obj_meta.rs

```diff
@@ -32,7 +32,7 @@ pub struct ObjectMeta {
 }
 
 impl ObjectMeta {
-    pub fn get_name(&self) -> String {
+    pub fn _get_name(&self) -> String {
```

Comment:
Please remind me why this needs to happen.

Reply:
Just to disable the Clippy warning about unused functions. We could also completely remove it.